### ISSUE TITLE
Update Images Docs

### ISSUE THREAD

**Issue by steveperry-53** [kubernetes/website#4502]
**This is a...** 

- [x] Feature Request
- [ ] Bug Report

Study this topic: [Images](https://kubernetes.io/docs/concepts/containers/images/). Either delete this topic or prune it. Much of the content here is specific to cloud providers, and should be covered in the various cloud provider docs.

**Comment by sftim** [comment]
/priority backlog
Is that OK?

**Comment by sftim** [comment]
Increasing priority to 
/remove-priority backlog
/priority important-longterm

**Comment by sftim** [comment]
Here are some things the reader should learn from the concept page:
- images are artifacts
- images represent runnable applications
- images have layers
- images have content-addressable identifiers (hashes)
- https://github.com/opencontainers/image-spec specifies how images work
- there are also [manifests](https://github.com/opencontainers/image-spec/blob/master/manifest.md) that work like images in some ways but aren't the same thing

[IMO] signposting readers to further reading is fine where appropriate.

**Comment by sftim** [comment]
Also see https://github.com/kubernetes/website/issues/40664 and https://github.com/kubernetes/website/issues/40530

**Comment by a-mccarthy** [comment]
@sftim I'd be willing to work on this, but i'm hoping you can share more of your thoughts on whats needed to be added or removed since the issue summary was from a while ago and the page has been updated a bunch since this issue was created. I see your comment, 

> Here are some things the reader should learn from the concept page:
> 
> * images are artifacts
> * images represent runnable applications
> * images have layers
> * images have content-addressable identifiers (hashes)
> * https://github.com/opencontainers/image-spec specifies how images work
> * there are also [manifests](https://github.com/opencontainers/image-spec/blob/master/manifest.md) that work like images in some ways but aren't the same thing
> 
> [IMO] signposting readers to further reading is fine where appropriate.

Is that all you recommended improving? Are there another folks I should reach out to for improvement idea for this page? Thanks!

https://github.com/kubernetes/website/issues/39870 also seems related to this work.

**Comment by sftim** [comment]
I'm confident that the changes you'd be proposing would be strongly net beneficial @a-mccarthy - please go with what feels right to you, and don't feel obliged to cover everything we _could_ feasible document here.

You could pick one of those bullet points and try to cover what aspect of that you feel comfortable writing about. You don't even have to round off the whole subtopic.

Small PRs that make the page a little better are ideal because we have very few contributors willing to review big PRs.

For extra help, CNCF techdocs might be willing to introduce you to other tech writers who'd be interested in the topic.

**Comment by sftim** [comment]
I agree, https://github.com/kubernetes/website/issues/39870 is related too.

**Comment by sergeyshevch** [comment]
I want to work on this issue. As I see some points above are already addressed. 

I will introduce a few small PR's in the next weeks with improvements for this page

/assign

**Comment by sftim** [comment]
PRs are (still) welcome.

**Comment by lmktfy** [comment]
What's left to do?
I think the next steps should be:
 
- for each improvement, [originally](https://github.com/kubernetes/website/issues/4502#issue-245814606) covered, that still needs doing: open a new issue per specific improvement
- close _this_ issue

**Comment by AresPhoenix345** [comment]
@sergeyshevch 
Any updates on this issue?
